### PR TITLE
add reading function for py6s results in PIU.py

### PR DESCRIPTION
**Pull request by AlexisDeru**
This commit will allow Ashley to read the py6s dataset a L2

### CODE REVIEW

File: Source/ProcessL1b_FRMCal.py
Comment (oceancolorcoder) [inline Source/ProcessL1b_FRMCal.py:1]: Not clear on these changes, swapping out one commented block for another.

File: Source/ProcessL1bqc.py
Comment (oceancolorcoder) [inline Source/ProcessL1bqc.py:1]: If L701 is for debugging it should be commented out later

File: Source/ProcessL1b_FRMCal.py
Comment (AlexisDeru) [reply Source/ProcessL1b_FRMCal.py:1]: I removed a previous commented block to clean the code (running py6s only one per cast)

The new commented block is actually the 'py6s_full' group creation, It seems that it is not needed but I want to keep trace of it. I indend to remove this block if we can work with the existing py6s group

File: Source/ProcessL1bqc.py
Comment (AlexisDeru) [reply Source/ProcessL1bqc.py:1]: yes you are right, sorry for that, will do.

**Pull request merged**